pkg/xds/envoy/listeners/v3/retry_configurer.go

```diff
@@ -93,6 +93,21 @@ func genHttpRetryPolicy(
 		policy.RetriableStatusCodes = conf.RetriableStatusCodes
 	}
 
+	if conf.RetriableMethods != nil {
+		for _, methodValue := range conf.RetriableMethods {
```

Comment:
You can drop the nil checks and just do this:
```Go
for _, methodValue := range conf.GetRetriableMethods() {
        ...
}
```

Reply:
Good idea. I didn't know this function.